test/runtests.jl

```diff
@@ -368,6 +379,11 @@ end
     roundtripped = Legolas.read(Legolas.tobuffer(tbl, NestedAgainV1SchemaVersion()))
     @test roundtripped.n[1] == NestedV1(; gc=GrandchildV1(r0_roundtripped), k="test")
     @test roundtripped.h[1] == 3
+
+    @testset "docstring support" begin
+        ds = Docs.docstr(Docs.Binding(@__MODULE__, :DocumentedV1))
```

Comment:
You could also get the docstring in a slightly less scary way with `string(@doc DocumentedV1)`

Reply:
Thanks!